lang/en/local_boostnavigation.php

```diff
@@ -117,6 +117,8 @@
 <li><b>Supported language(s) (optional):</b> This setting can be used for displaying the custom node to users of the specified language only. Separate more than one supported language with commas. If the custom node should be displayed in all languages, then leave this field empty.</li >
 <li><b>Supported cohort(s) (optional):</b> This setting can be used for displaying the custom node to members of the specified cohort only. Use the cohort\'s ID, not the cohort\'s name, for this setting. Separate more than one supported cohort with commas. If the custom node should be displayed for users regardless of any cohort membership, then leave this field empty.</li>
 <li><b>Supported role(s) (optional):</b> This setting can be used for displaying the custom node only to members with the specified role in each context. Use the role\'s shortname for this setting. Separate more than one supported role with commas. If the custom node should be displayed for users regardless of any role, then leave this field empty.</li>
+<li><b>Supported global role(s) (optional):</b> This setting can be used for displaying the custom node only to users with the specified global role. Use the role\'s shortname for this setting. Separate more than one supported role with commas. If the custom node should not be displayed for users with global role(s), then leave this field empty.</li>
```

Comment:
Currently, the 5th parameter only processes roles which are assigned on the course level. So, this addition for system roles generally makes sense for some setups and I am happy to add it.

There are two things which I would like to mention:
* Please let's use the term "system roles" and not "global roles" as this is the term used in Moodle core on /admin/roles/assign.php?contextid=1
* Your sentence "If the custom node should not be displayed for users with global role(s), then leave this field empty." is wrong. The node is added to the navigation unless it is restricted to certain roles. Thus, you don't hide the node if this field is empty. Please rephrase it as "If the custom node should be displayed for users regardless of any system role, then leave this field empty." which is then the same as for the fifth parameter.